shardingsphere-features/shardingsphere-master-slave/shardingsphere-master-slave-route/src/test/java/org/apache/shardingsphere/masterslave/route/engine/MasterSlaveRouteDecoratorTest.java

```diff
@@ -148,7 +158,17 @@ public void assertDecorateToMasterWithoutRouteUnitsAndWithParameters() {
         assertThat(routedDataSourceNames.next(), is(MASTER_DATASOURCE));
         assertThat(actual.getParameters().get(0), is("true"));
     }
-    
+
+    @Test
+    public void assertTypeClass() {
+        assertThat(routeDecorator.getTypeClass(), org.hamcrest.core.Is.<Class<?>>is(MasterSlaveRule.class));
+    }
+
+    @Test
+    public void assertOrder() {
```

Comment:
It is the static code, so the test case is unnecessary.

Reply:
I misunderstood.
delete it already. thx!